src/main/java/com/github/alex1304/jdash/util/UserRankingStrategy.java

```diff
@@ -0,0 +1,18 @@
+package com.github.alex1304.jdash.util;
+
+public enum UserRankingStrategy {
```

Comment:
Rename to `LeaderboardSortingStrategy`

Reply:
Nevermind, "sorting" isn't really relevant for friends, because it's a filter and not a sorting. But idk `LeaderboardStrategy` sounds weird, because leaderboard isn't an action but an entity. `LeaderboardType` is probably better.

Reply:
Changed to _LeaderboardType_
also Renamed param _strategy_ to _type_ for _GDLevelCommentsRequest_